.github/workflows/release-prepare.yml

```diff
@@ -241,4 +227,4 @@ jobs:
           ## limits ssh access and adds the ssh public key for the user which triggered the workflow
           limit-access-to-actor: true
           ## limits ssh access and adds the ssh public keys of the listed GitHub users
-          limit-access-to-users: steveeJ,jost-s,freesig,neonphog,thedavidmeister,maackle
```

Comment:
An oopsie, this shouldn't still have been here and once I saw it I figured I'd grant myself access.